docs/usage/configuration-options.md

```diff
@@ -1676,7 +1677,10 @@ Any PR that is being updated will be automerged with the Renovate-based automerg
 This option is available for Azure, GitHub and GitLab.
 It falls back to Renovate-based automerge if the platform-native automerge is not available.
 
-Though this option is enabled by default, you can fine tune the behavior by setting `packageRules` if you want to use it selectively (e.g. per-package).
+You can also fine-tune the behavior by setting `packageRules` if you want to use it selectively (e.g. per-package).
+
+If you rely on Renovate's behavior of rebasing automerging branches when `rebaseWhen=auto`, then note that you are no longer guaranteed of up-to-date branches when automerging if you enable platform automerge.
```

Comment:
```suggestion
Note that `rebaseWhen=auto` and `platformAutomerge=true` are not fully compatible.
Normally when you set `rebaseWhen=auto` Renovate rebases any branch that's behind the base branch automatically.
This behavior is no longer guaranteed when you enable `platformAutomerge`.
```

How about something like this?

Not sure if the text matches the code/behavior, so check before applying this suggestion.